Editor/Scripts/demo_tutorial.py

```diff
@@ -5,7 +5,7 @@
 SPDX-License-Identifier: Apache-2.0 OR MIT
 """
 
-from PySide2.QtWidgets import QMenuBar
+from PySide2.QtWidgets import QMenuBar, QMainWindow
```

Comment:
Nitpick: It doesn't look like this `QMainWindow` import gets used

Reply:
fixed